examples/meson.build

```diff
@@ -23,3 +23,8 @@ executable(
     link_with: libnvme,
     include_directories: incdir)
 
+executable(
+    'mi-mctp',
+    ['mi-mctp.c'],
+    link_with: libnvme_mi_static,
```

Comment:
This looks wrong. I think you should just use libnvme_mi here, the user should be able to choose how the library is build/used.

Reply:
Ack, will fix.